portfolio/src/main/webapp/style.css

```diff
@@ -64,4 +64,6 @@ h4 {text-align: center; font-family: Verdana, serif;}
   height: 70px;
   text-align: start;
   background-color: snow;
-}
\ No newline at end of file
+}
+
+.hidden {display: none;}
```

Comment:
is this formatting consistent with the rest of your file? I think block indentation should be used https://google.github.io/styleguide/htmlcssguide.html#Block_Content_Indentation

Reply:
I have some other one-line formatting. But thank you for point out the website! It makes sense. I will change them all to the format with block content indentation.